src/Native/Bootstrap/common.h

```diff
@@ -70,6 +70,7 @@ struct RawEEType
     uint16_t    m_usNumVtableSlots;
     uint16_t    m_usNumInterfaces;
     uint32_t    m_uHashCode;
+	void*   m_indirectionModule;
```

Comment:
Nit: can you align it with the rest of the members (add 5 spaces) and prefix with `p` (`m_pIndirectionModule`)?